tests/Jenkins/utilities.sh

```diff
@@ -332,6 +332,26 @@ function diracInstallCommand(){
 	./dirac-install -r `cat dirac.version` -t server -d
 }
 
+#.............................................................................
+#
+# prepareForServer:
+#
+#   This function gets the DIRAC install script
+#
+#.............................................................................
+
+function prepareForServer(){
+	echo '==> [prepareForServer]'
+
+	#Move to server directory
+	cd $SERVERINSTALLDIR
```

Comment:
I see, it's a fair point. I removed the navigation from this function and from others, but in some I could not.